--- neofetch	2024-05-14 22:14:21.192273209 +0100
+++ neofetch.modified	2024-05-14 22:30:22.752275832 +0100
@@ -1641,9 +1641,18 @@
             has snap && ps -e | grep -qFm 1 snapd >/dev/null && \
             pkgs_h=1 tot snap list && ((packages-=1))
 
-            # This is the only standard location for appimages.
+            # AppImages installed in ~/Applications.
             # See: https://github.com/AppImage/AppImageKit/wiki
-            manager=appimage && has appimaged && dir ~/.local/bin/*.appimage
+            #manager=AppImage && dir ~/Applications/*.appimage
+
+            # Cargo packages
+            #manager=Cargo && dir ~/.cargo/bin/*
+
+            # FromSrc
+            #manager=FromSrc && dir ~/.s/*
+
+            # SImple Package manager
+            manager=SIP && sip count
         ;;
 
         "Mac OS X"|"macOS"|MINIX)
